README

```diff
@@ -294,6 +301,15 @@ The default behaviour can be changed with the following directives:
 
 	Default: 5000 (== 5 s).
 
+    LookupArgumentName
```

Comment:
Could we have "Cert" somewhere in the directive name so that it's clearer what functionality it refers to?